### PR TITLE
added media queries tutorial and assets

### PR DESCRIPTION
**Pull request by thisisjofrank**
New media queries and responsive design tutorial for the HTML tutorials section.

### CODE REVIEW

File: html/lesson7/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+    <footer class="footer">
+    </footer>
+
+This is the structure of our page.
+
+###Add in the content
+
+Now we're ready to flesh out our structure with content.  Looking at the example page, starting from the top we can see that the header contains an image and a heading, so lets put them into our HTML, with class names
+
+    <img class="header_img" src="assets/images/gracehopper_portrait.jpg" />
```
Comment (KimberleyCook) [inline html/lesson7/tutorial.md]: can remove the / from the end of the img element please. I removed them all in a previous PR as that is old HTML so to keep it consistent throughout all tutorials.

File: html/lesson7/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+    <h1 class="header_title"> Grace Hopper</h1>
+
+Next we have the copy in paragraphs interspersed with figures and a blockquote. Add this to your about section using
+
+    <p class="main_text">
+        ...
+    </p>
+
+    <figure class="figure">
+        <img class="figure_img" src="assets/images/grace.jpg">
```
Comment (KimberleyCook) [inline html/lesson7/tutorial.md]: can we make sure all images in this tutorial have alt tags please. We say it's really important for accessibility in tutorial 2 so again for consistency.

File: html/lesson7/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+        <li class="resourcelist_item">
+            <a class="resourcelist_link" href="#">Computer Scientist</a>
+        </li>
+        ...
+    </ul>
+
+
+And finally we have the footer which will contain the author information. This will just be a p tag with an a. Give them appropriate class names eg
+
+    <p class="footer_text">Made by <a href="http://codebar.io" class="footer_link">codebar.io</a></p>
```
Comment (KimberleyCook) [inline html/lesson7/tutorial.md]: in the above a element you have put the class="blah" after the href="blah" and here the other way round, could they be consistent please. This is small but could easily confuse a student. Thanks.

File: html/lesson7/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+What we need is a way to add styles that will only be applied if the browser is above a certain width.
+
+The way we do this is with css <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries" target="_blank">media queries</a>.
+
+###Media Queries
+A media query lets you restrict your css to only be applied when the condition of the query is true, for example when the media type specified in the media query matches the device that the user is viewing your site on.
+You may have seen them used to swap styles on a page for screen and for print eg
+
+    <link rel="stylesheet" media="screen" href="style.css" />
+    <link rel="stylesheet" media="print" href="print.css" />
```
Comment (KimberleyCook) [inline html/lesson7/tutorial.md]: again can we remove the / from the end of both of these please.

File: html/lesson7/tutorial.md (hunk 3 above)
Comment (thisisjofrank) [inline html/lesson7/tutorial.md]: resolved

Comment (KimberleyCook) [inline html/lesson7/tutorial.md]: @thisisjofrank happy for me to merge?

Comment (thisisjofrank) [inline html/lesson7/tutorial.md]: Yep! Exciting! :D

**Comment by KimberleyCook** [comment]
all merged :D thank you so much Jo

**Pull request merged**